README.md

```diff
@@ -0,0 +1,41 @@
+## Screens
+
+### Development
+
+#### Prerequisites
+- [Docker](https://www.docker.com/docker-toolbox)
```

Comment:
Am accepting all the defaults. It is installing to Macintosh HD. Hope that's ok.

Which "tool to get started with Docker"? (going with Docker Quickstart Terminal)
It popped up a new terminal window (or used an open one) and displayed this:
Last login: Mon Dec 14 13:57:26 on ttys000
FT-MW4451:screens chris.gathercole$ bash --login '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'





















Creating Machine default...
Creating CA: /Users/chris.gathercole/.docker/machine/certs/ca.pem
Creating client certificate: /Users/chris.gathercole/.docker/machine/certs/cert.pem
Running pre-create checks...
Creating machine...
(default) Creating VirtualBox VM...
(default) Creating SSH key...
(default) Starting VM...
Waiting for machine to be running, this may take a few minutes...
Machine is running, waiting for SSH to be available...
Detecting operating system of created instance...
Detecting the provisioner...
Provisioning with boot2docker...
Copying certs to the local machine directory...
Copying certs to the remote machine...
Setting Docker configuration on the remote daemon...
Checking connection to Docker...
Docker is up and running!
To see how to connect Docker to this machine, run: /usr/local/bin/docker-machine env default
Setting environment variables for machine default...






                        ##         .
                  ## ## ##        ==
               ## ## ## ## ##    ===
           /"""""""""""""""""\___/ ===
      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~~ ~ /  ===- ~~~
           \______ o           __/
             \    \         __/
              \____\_______/


docker is configured to use the default machine with IP 192.168.99.100
For help getting started, check out the docs at https://docs.docker.com

FT-MW4451:screens chris.gathercole$